### PR TITLE
User can View Profile

### PR DESCRIPTION
**Pull request by clalil**
# User can View Profile
[PT-link](https://www.pivotaltracker.com/story/show/169861930)

## This PR Contains
- A basic User Profile
- Ability to read and be directed to own recipes
- Feature test for happy and sad paths

Screenshot 2019-11-20 at 18 22 33

### CODE REVIEW

File: cypress/integration/userCanViewOwnRecipesInProfile.spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      status: 200
+    })
+
+    cy.loginUser('user@mail.com', 'password')
+    cy.get('#navbar')
+      .within(() => {
+        cy.get('#nav-profile').click()
+      })
+    cy.get('#message')
+      .should('contain', 'After you have created a recipe you can always access it from here')
```
Comment (beccaburns) [inline cypress/integration/userCanViewOwnRecipesInProfile.spec.js]: Can this message be more specific - "You do not currently have any recipes."

File: src/components/UserProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <Link id={`recipe-${recipe.id}`} to={`/recipe/${recipe.id}`}><List.Description className="profile-link">View recipe</List.Description></Link>
+              </List.Content>
+          </List.Item>
+        )
+      })
+    }
+
+    message = (
+      <Header as="p" id="message" style={{ color: "#4C5966", textAlign: 'center' }}>
+        After you have created a recipe you can always access it from here
```
Comment (beccaburns) [inline src/components/UserProfile.jsx]: Same as above. Just an idea, but I think it is more direct.

**Review by beccaburns — DISMISSED** [review]

File: src/components/UserProfile.jsx (hunk 2 above)
Comment (clalil) [reply src/components/UserProfile.jsx]: Good idea! Will do ! :)

File: cypress/integration/userCanViewOwnRecipesInProfile.spec.js (hunk 1 above)
Comment (clalil) [reply cypress/integration/userCanViewOwnRecipesInProfile.spec.js]: Good idea! Will do! :)

**Review by shsteindorsson — APPROVED** [review]

**Review by yasmineezequiel — APPROVED** [review]

**Review by miyeaier — APPROVED** [review]

**Pull request merged**